test/proxysql/README.md

```diff
@@ -0,0 +1,77 @@
+# ProxySQL in Boulder
+
+In an effort to keep Boulder's development environment reasonably close to
+production we use ProxySQL in our Docker stack to proxy connections to our
+MariaDB database.
+
+## Ports
+
+ProxySQL listens on the following ports:
+  - `6033` Proxy MySQL Interface
+  - `6032` Admin MySQL Interface
+  - `6080` Admin Web Interface
+
+## Accessing the Admin MySQL Interface
+
+```bash
+mysql -uradmin -pradmin -h 127.0.0.1 --port 6032
+```
+
+### MacOS
+
+You will need to bind the port in `docker-compose.yml`, like so:
+
+```yaml
+  bproxysql:
+    ports:
+      - 6032:6032
```

Comment:
Maybe include this commented out in docker-compose.yml if we don't want it enabled all the time?

Reply:
Stylistically we've typically steered clear of leaving commented config in our docker-compose file. Also, this is only required for folks running MacOS.